src/main/java/core/basesyntax/operation/impl/BalanceOperationHandler.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.operation.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.operation.OperationHandler;
+
+public class BalanceOperationHandler implements OperationHandler {
+    private FruitDao fruitDao;
```

Comment:
you should use here service layer, not dao, so create a FruitService and inside of it you can add field FruitDao and use its methods, the same for other classes